lib/apirequest.js

```diff
@@ -184,6 +186,10 @@ function createAPIRequest(parameters, callback) {
   delete options.auth; // is overridden by our auth code
   delete options.params; // We handle params ourselves and Request does not recognise 'params'
 
+  if (missingCallback) {
```

Comment:
Was originally going to do that but the createCallback function guarantees that !callback will always evaluate to false